src/renderer/state/flipper.ts

```diff
@@ -66,6 +112,14 @@ type FlipperMode =
   | 'front_disable'
   | 'back_enable'
   | 'back_disable'
+  | 'fr_enable'
+  | 'fr_disable'
+  | 'fl_enable'
+  | 'fl_disable'
+  | 'br_enable'
```

Comment:
Hummm in Markhor's code, the flippers that are on the back are supposed to be called rear_<left or right>.

Reply:
Yes good catch :) This was changed locally after noticing it during testing. Pushing now.